src/agent/src/config.rs

```diff
@@ -26,6 +26,7 @@ const LOG_VPORT_OPTION: &str = "agent.log_vport";
 const CONTAINER_PIPE_SIZE_OPTION: &str = "agent.container_pipe_size";
 const UNIFIED_CGROUP_HIERARCHY_OPTION: &str = "agent.unified_cgroup_hierarchy";
 const CONFIG_FILE: &str = "agent.config_file";
+const REST_API_OPTION: &str = "agent.guest_components_rest_api";
```

Comment:
@stevenhorsman We could consider adding the option to [README](https://github.com/kata-containers/kata-containers/blob/main/src/agent/README.md#:~:text=tool%20documentation.-,Agent%20options,-The%20kata%20agent) for easier user comprehension. In my opinion, I think the PR is good enough and this comment should not block the merging. It could be addressed in a follow up PR.

Reply:
Thanks for the reminder, I'll add that as part of this PR